### PR TITLE
Merge 17.7 into develop after 17.6.1, 17.7-rc-2 and Release Notes

### PR DESCRIPTION
**Pull request by AliSoftware**
This includes:

 - Release notes from Editorial, both for WordPress and Jetpack (see p2JRYi-5oG-p2#comment-20626 for WP and reply for JP)
 - Update of our Fastfile to generate the `JetpackPlayStoreStrings.po` file from the Jetpack `release_notes.txt` files
 - The fix https://github.com/wordpress-mobile/WordPress-Android/pull/14960 (toggling feature flag that was left off before code freeze)

🚨  Note: For consistency in the git branch graph, [the (empty) PR to merge hotfix 17.6.1 into develop](https://github.com/wordpress-mobile/WordPress-Android/pull/14971) should be merged before this one.

### CODE REVIEW

File: WordPress/jetpack_metadata/PlayStoreStrings.po [hunk 1]
```diff
@@ -0,0 +1 @@
+../metadata/JetpackPlayStoreStrings.po
```
Comment (AliSoftware) [inline WordPress/jetpack_metadata/PlayStoreStrings.po:1]: A symlink for convenience until we update the cron job in the sandbox to point to the new location 😓

File: WordPress/metadata/JetpackPlayStoreStrings.po [hunk 2]
```diff
@@ -11,6 +11,12 @@ msgstr ""
 "Project-Id-Version: Jetpack - Apps - Android - Release Notes\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_177"
+msgid ""
+"17.7:\n"
+"Block editor: New block layout grid, Badge style tweaks, Fixed new block not created when pressing enter inside text-based block, Tablet view fixes for inserter button\n"
```
Comment (AliSoftware) [inline WordPress/metadata/JetpackPlayStoreStrings.po:17]: p2JRYi-5oG-p2#comment-20627

File: WordPress/metadata/PlayStoreStrings.po [hunk 3]
```diff
@@ -11,6 +11,15 @@ msgstr ""
 "Project-Id-Version: Release Notes & Play Store Descriptions\n"
 
 #. translators: Release notes for this version to be displayed in the Play Store. Limit to 500 characters including spaces and commas!
+msgctxt "release_note_177"
+msgid ""
+"17.7:\n"
+"New! Get blogging reminder prompts on days of your choice\n"
+"Block editor: New block layout grid, Badge style tweaks, Fixed new block not created when pressing enter inside text-based block, Tablet view fixes for inserter button\n"
+"Fixed Reader duplicate post issue\n"
+"Added ability to subscribe to conversations by email from Reader\n"
```
Comment (AliSoftware) [inline WordPress/metadata/PlayStoreStrings.po]: p2JRYi-5oG-p2#comment-20626

File: version.properties [hunk 4]
```diff
@@ -1,12 +1,12 @@
-#Mon, 28 Jun 2021 15:56:12 +0200
+#Thu, 01 Jul 2021 17:14:55 +0200
 
 # WordPress version information
-wordpress.versionName=17.7-rc-1
-wordpress.versionCode=1068
+wordpress.versionName=17.7-rc-2
+wordpress.versionCode=1072
```
Comment (AliSoftware) [inline version.properties:5]: The jump is due to the 17.6.1 hotfix that was done in-between

**Review by oguzkocer — APPROVED** [review]

**Pull request merged**